.github/workflows/builds.yml

```diff
@@ -8,33 +8,41 @@ on:
 
 env:
   CTEST_OUTPUT_ON_FAILURE: 1
-  
+
 jobs:
   builds:
     name: ${{ matrix.platform.name }}-${{ matrix.build }}
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/ubuntu2004:v11
+    container: ${{ matrix.platform.container }}
     strategy:
       matrix:
         platform:
           - name: CPU
             container: ghcr.io/acts-project/ubuntu2004:v11
           - name: CUDA
-            container: ghcr.io/acts-project/ubuntu1804_cuda:v11
+            container: ghcr.io/acts-project/ubuntu2004_cuda:v13
         build:
           - Release
           - Debug
     steps:
       - name: Dependencies
         run: apt-get install -y git-lfs
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v1      
         with:
           submodules: true
           lfs: true
       - name: Configure
-        run: cmake -DCMAKE_BUILD_TYPE=${{ matrix.build }} -B build -S .
+        run: >
+          if ${{ matrix.platform.name == 'CPU'}}; then
+            cmake -B build -S . \
+              -DCMAKE_BUILD_TYPE=${{ matrix.build }} 
+          elif ${{ matrix.platform.name == 'CUDA'}}; then
+            cmake -B build -S . \
+              -DCMAKE_BUILD_TYPE=${{ matrix.build }} \
+              -DCMAKE_CUDA_COMPILER=/usr/local/cuda/bin/nvcc \
```

Comment:
I would recommend `$(which nvcc)` here to make the finding a little more robust if we add more platforms later.